test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2236,6 +2236,146 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
             $pathologicalRatio | Should -BeGreaterThan 5
         }
     }
+
+    Context 'Invoke-WebSession: Connection persistence in a WebSession' {
+        $connWarning = [regex]::new('WebSession.+HTTP')
+
+        function RunCheckingPersistence {
+            param(
+                [uri]$Uri,
+                [string]$Command,
+                [object]$Session,
+                [switch]$LostPersistence,
```

Comment:
Maybe 
```suggestion
                [switch]$ExpectLostPersistence,
```

Reply:
Used `-ExpectConnectionRecreated` for the function parameter to better match the resource string name